libs/common/package.json

```diff
@@ -24,7 +24,8 @@
     "./change-stream": "./src/change-stream/index.ts",
     "./change-stream/test-helpers": "./src/change-stream/test-helpers/index.ts",
     "./payload-verification": "./src/helpers/payload-verification/index.ts",
-    "./schemas": "./src/schemas/index.ts"
+    "./schemas": "./src/schemas/index.ts",
+    "./test-cases": "./src/test-helpers/test-cases/index.ts"
```

Comment:
is test cases descriptive enough? This is isolated because it uses mongodb so that should be made clearer?

Reply:
I think it's clear enough. You don't want a separate export for each test case we define (I'm planning on adding one for SQL id validation), and I can't think of a way of naming this which narrows the description without narrowing what we can include in the export. Did you have any other names in mind?

Reply:
we could do `test-cases-backend` and `test-cases-frontend` and `test-cases` or even `test-cases-using-mongo-db`?... but I feel that might add confusion or is clunky and not very nice? Would this make using this for front end tests annoying as we'd have to update to be able to handle this?

To be honest this question has come up a few times now... I think every time we take the `best for now` solution, and here I wouldn't mind doing the same...

Reply:
I have decided that I don't need the `ObjectId` for this, so I should be able to include it in the main export

Reply:
Turns out `supertest` also is only used in backend so I have changed the export to `test-cases-backend`